### PR TITLE
Updated Encoding parameter and YAML for Format-Hex

### PR DESCRIPTION
**Pull request by davidsmatlak**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [x] The documented feature was introduced in versions 5.0, 5.1, and 6 of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

Related to Issue #2000
Updated Encoding parameter for Format-Hex
Style updates
Paragraph reflows
Fixed links

Acrolinx scores
Original version: 88
Updated version: 93

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[717ea60](https://github.com/davidsmatlak/PowerShell-Docs/pull/3649/commits/717ea6077d96d8042997fc36d2a962c1ae481f2d)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[717ea60](https://github.com/davidsmatlak/PowerShell-Docs/pull/3649/commits/717ea6077d96d8042997fc36d2a962c1ae481f2d)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 52 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[717ea60](https://github.com/davidsmatlak/PowerShell-Docs/pull/3649/commits/717ea6077d96d8042997fc36d2a962c1ae481f2d)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-formathex/reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/format-hex?view=powershell-5.0&branch=pr-en-us-3649) (powershell-5.0) | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Utility%2FFormat-Hex.md)
[reference/5.1/Microsoft.PowerShell.Utility/Format-Hex.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-formathex/reference/5.1/Microsoft.PowerShell.Utility/Format-Hex.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/format-hex?view=powershell-5.1&branch=pr-en-us-3649) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Utility%2FFormat-Hex.md)
[reference/6/Microsoft.PowerShell.Utility/Format-Hex.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-formathex/reference/6/Microsoft.PowerShell.Utility/Format-Hex.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/format-hex?view=powershell-6&branch=pr-en-us-3649) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Utility%2FFormat-Hex.md)

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Utility%2FFormat-Hex.md"></a>
### [reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-formathex/reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md)
 - **[Warning]** Invalid file link:(~/5.0/Microsoft.Powershell.Core/About/about_quoting_rules).

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Utility%2FFormat-Hex.md"></a>
### [reference/5.1/Microsoft.PowerShell.Utility/Format-Hex.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-formathex/reference/5.1/Microsoft.PowerShell.Utility/Format-Hex.md)
 - **[Warning]** Invalid file link:(~/5.1/Microsoft.Powershell.Core/About/about_quoting_rules).

<a name="reference%2F6%2FMicrosoft.PowerShell.Utility%2FFormat-Hex.md"></a>
### [reference/6/Microsoft.PowerShell.Utility/Format-Hex.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-formathex/reference/6/Microsoft.PowerShell.Utility/Format-Hex.md)
 - **[Warning]** Invalid file link:(~/6/Microsoft.Powershell.Core/About/about_quoting_rules).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C5%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902050145411908-3649%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=JBzOSuOzjumJKZAhpAg%2FLOaNSPmFcG%2FW1ZdIDRTg8AQ%3D&st=2019-02-05T01%3A54%3A24Z&se=2019-03-08T01%3A59%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md [hunk 1]
```diff
@@ -148,25 +176,32 @@ Accept wildcard characters: False
...
 
 ### Microsoft.PowerShell.Commands.ByteCollection
-This cmdlet returns a **ByteCollection**.
-This object represents a collection of bytes.
-It includes methods that convert the collection of bytes to a string formatted like each line of output returned by **Format-Hex**.
-If you specify the *Path* or *LiteralPath* parameter, the object also contains the path of the file that contains each byte.
+
+This cmdlet returns a **ByteCollection**. This object represents a collection of bytes. It includes
+methods that convert the collection of bytes to a string formatted like each line of output returned
+by **Format-Hex**. If you specify the *Path* or *LiteralPath* parameter, the object also contains
```
Comment (sdwheeler) [inline reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md]: `*Path* or *LiteralPath* parameter` - should be bold for parameters

File: reference/5.1/Microsoft.PowerShell.Utility/Format-Hex.md [hunk 2]
```diff
@@ -163,25 +198,32 @@ Accept wildcard characters: False
...
 
 ### Microsoft.PowerShell.Commands.ByteCollection
-This cmdlet returns a **ByteCollection**.
-This object represents a collection of bytes.
-It includes methods that convert the collection of bytes to a string formatted like each line of output returned by **Format-Hex**.
-If you specify the *Path* or *LiteralPath* parameter, the object also contains the path of the file that contains each byte.
+
+This cmdlet returns a **ByteCollection**. This object represents a collection of bytes. It includes
+methods that convert the collection of bytes to a string formatted like each line of output returned
+by **Format-Hex**. If you specify the *Path* or *LiteralPath* parameter, the object also contains
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Utility/Format-Hex.md]: ` *Path* or *LiteralPath* parameter` - should be bold for parameters

File: reference/6/Microsoft.PowerShell.Utility/Format-Hex.md [hunk 3]
```diff
@@ -192,25 +235,32 @@ Accept wildcard characters: False
...
 
 ### Microsoft.PowerShell.Commands.ByteCollection
-This cmdlet returns a **ByteCollection**.
-This object represents a collection of bytes.
-It includes methods that convert the collection of bytes to a string formatted like each line of output returned by **Format-Hex**.
-If you specify the *Path* or *LiteralPath* parameter, the object also contains the path of the file that contains each byte.
+
+This cmdlet returns a **ByteCollection**. This object represents a collection of bytes. It includes
+methods that convert the collection of bytes to a string formatted like each line of output returned
+by **Format-Hex**. If you specify the *Path* or *LiteralPath* parameter, the object also contains
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Utility/Format-Hex.md]: ` *Path* or *LiteralPath* parameter` - should be bold for parameters

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[6d4ba7e](https://github.com/davidsmatlak/PowerShell-Docs/pull/3649/commits/6d4ba7e008c766b86bbdc17daca51508eb56431a)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[6d4ba7e](https://github.com/davidsmatlak/PowerShell-Docs/pull/3649/commits/6d4ba7e008c766b86bbdc17daca51508eb56431a)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 52 sec(s)_

File: reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md (hunk 1 above)
Comment (davidsmatlak) [reply reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md]: Thanks. Updated the formatting and submitted the changes for review.

File: reference/6/Microsoft.PowerShell.Utility/Format-Hex.md (hunk 3 above)
Comment (davidsmatlak) [reply reference/6/Microsoft.PowerShell.Utility/Format-Hex.md]: Thanks. Updated the formatting and submitted the changes for review.

File: reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md [hunk 4]
```diff
@@ -108,13 +139,11 @@ Accept wildcard characters: False
 ```
 
 ### -LiteralPath
```
Comment (davidsmatlak) [inline reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md]: Revised and corrected content for the parameters LiteralPath and Path.

File: reference/5.1/Microsoft.PowerShell.Utility/Format-Hex.md [hunk 5]
```diff
@@ -109,13 +140,11 @@ Accept wildcard characters: False
 ```
 
 ### -LiteralPath
```
Comment (davidsmatlak) [inline reference/5.1/Microsoft.PowerShell.Utility/Format-Hex.md]: Revised and corrected content for the parameters LiteralPath and Path.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[6d4ba7e](https://github.com/davidsmatlak/PowerShell-Docs/pull/3649/commits/6d4ba7e008c766b86bbdc17daca51508eb56431a)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-formathex/reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/format-hex?view=powershell-5.0&branch=pr-en-us-3649) (powershell-5.0) | [Details](#user-content-reference%2F5.0%2FMicrosoft.PowerShell.Utility%2FFormat-Hex.md)
[reference/5.1/Microsoft.PowerShell.Utility/Format-Hex.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-formathex/reference/5.1/Microsoft.PowerShell.Utility/Format-Hex.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/format-hex?view=powershell-5.1&branch=pr-en-us-3649) (powershell-5.1) | [Details](#user-content-reference%2F5.1%2FMicrosoft.PowerShell.Utility%2FFormat-Hex.md)
[reference/6/Microsoft.PowerShell.Utility/Format-Hex.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-formathex/reference/6/Microsoft.PowerShell.Utility/Format-Hex.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.utility/format-hex?view=powershell-6&branch=pr-en-us-3649) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Utility%2FFormat-Hex.md)

<a name="reference%2F5.0%2FMicrosoft.PowerShell.Utility%2FFormat-Hex.md"></a>
### [reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-formathex/reference/5.0/Microsoft.PowerShell.Utility/Format-Hex.md)
 - **[Warning]** Invalid file link:(~/5.0/Microsoft.Powershell.Core/About/about_quoting_rules).

<a name="reference%2F5.1%2FMicrosoft.PowerShell.Utility%2FFormat-Hex.md"></a>
### [reference/5.1/Microsoft.PowerShell.Utility/Format-Hex.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-formathex/reference/5.1/Microsoft.PowerShell.Utility/Format-Hex.md)
 - **[Warning]** Invalid file link:(~/5.1/Microsoft.Powershell.Core/About/about_quoting_rules).

<a name="reference%2F6%2FMicrosoft.PowerShell.Utility%2FFormat-Hex.md"></a>
### [reference/6/Microsoft.PowerShell.Utility/Format-Hex.md](https://github.com/davidsmatlak/PowerShell-Docs/blob/ds-issue2000-formathex/reference/6/Microsoft.PowerShell.Utility/Format-Hex.md)
 - **[Warning]** Invalid file link:(~/6/Microsoft.Powershell.Core/About/about_quoting_rules).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C5%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902051730488934-3649%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=4zEr7ezW%2BqEldImmpePPQE103OJHllK9Yawmw9eG%2F00%3D&st=2019-02-05T17%3A38%3A46Z&se=2019-03-08T17%3A43%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**